libs/ledger-live-common/src/api/Ethereum.ts

```diff
@@ -180,7 +182,7 @@ export const apiForCurrency = (currency: CryptoCurrency): API => {
         noinput: true,
       };
       if (blockHeight) {
-        query.block_height = blockHeight;
+        query.from_height = blockHeight;
```

Comment:
FYI @hedi-edelbloute

Reply:
Idk how this happened, either I got the attribute wrong or it got updated... anyway, it requires unit test update

Reply:
It could actually be a source of problem... if we only get 2000 txs, and some erc20 happen before those 2000txs